package.json

```diff
@@ -15,6 +15,7 @@
     "start:frontend:hot": "cd frontend && npm run start:hot",
     "start:frontend:launch": "cd frontend && npm run launch",
     "plugins": "concurrently npm:start:backend npm:serve:plugins npm:ocp-console npm:watch:multicluster-sdk -n backend,frontend,console,watch -c green,blue,magenta,bgCyan",
+    "plugins-ocp": "concurrently npm:start:backend npm:serve:plugins npm:ocp-console -n backend,frontend,consoleh -c green,blue,magenta",
```

Comment:
@jeswanke Was there a problem with running the watch?

Reply:
@KevinFCormier -- watch works great--I tried doing alias too but it was no faster then watch w/o the advantage of updating the types--the reason for 'plugins-ocp' is if I was doing a lot of editing and saving it would trigger the build before the code was all set--I just like saying "all done, create a build"